docs/dev/how_to/releasing_chef_infra.md

```diff
@@ -53,6 +56,8 @@ or for a previous release branch:
 
 `/expeditor promote chef/chef:chef-16 16.13.9`
 
+:warning: Do not `gem push` the ruby gem manually... this will prevent promotion of habitat packages to stable and will block notifications to chef.io slack channels.
+
```

Comment:
hmm, I thought the reason the next step failed (hab promote) was because the Linux and Linux2 packages had the same timestamp causing it to hit a weird edge case. It would also help to include how that can be manually triggered incase we do hit this blocker in the future.